src/hooks/useOverlay.tsx

```diff
@@ -29,6 +29,12 @@ export const useOverlay = ({
   const overlayRef = useProvidedRefOrCreate<HTMLDivElement>(_overlayRef)
   useOpenAndCloseFocus({containerRef: overlayRef, returnFocusRef, initialFocusRef, preventFocusOnOpen})
   useOnOutsideClick({containerRef: overlayRef, ignoreClickRefs, onClickOutside})
-  useOnEscapePress(onEscape)
+
+  /** We only want one overlay to close at a time */
```

Comment:
Nitpick: do we need a JSDoc-style comment here?
```suggestion
  // We only want one overlay to close at a time
```

Reply:
yep, not sure what I had here. fixed now.